themes/hugo-rladies/layouts/blog/single.html

```diff
@@ -1,45 +1,72 @@
 
-{{ define "main"}}
-    <div class="entry-content">
-      <p>{{ .Content }}</p>
-    </div>
-
-    {{ with .Params.tags -}}
-    
-      <div class="post-tags my-2">
-        {{ i18n "tags"}}:
-        {{ range . }}
-        {{- $name := . -}}
-        {{- with $.Site.GetPage "taxonomy" (printf "tags/%s" $name) | default ($.Site.GetPage "taxonomy" (printf "tags/%s" ($name | urlize))) -}}
-        <a href="{{ .Permalink }}">{{ $name }}</a>
-        {{ end -}}
-        {{ end }}
-      </div>
-    {{- end }}
+{{ define "title" }}
+<header class="entry-header">
```

Comment:
- I find the vertical space between authors and probably editors if there were several, too big.
- The left of the author's names is not aligned with the left of the word "Authors".

<img width="948" height="400" alt="image" src="https://github.com/user-attachments/assets/41256144-32c9-4365-a282-1fdf6864a78e" />

Reply:
Could the picture be placed higher up, or could it be higher?

Reply:
So we'd store two different versions of the picture, or two different pictures. That's what's done for tidyverse.org.

Reply:
can you elaborate on that last part a little?

Reply:
how do you like the newer version?

<img width="1844" height="1572" alt="Screenshot 2025-07-19 at 15 01 30" src="https://github.com/user-attachments/assets/0cf1fee4-490e-48a3-875a-7f273a4a1a06" />